cylc/flow/cfgspec/globalcfg.py

```diff
@@ -344,7 +345,8 @@
             ''')
             Conf('retrieve job logs', VDR.V_BOOLEAN, desc='''
                 Global default for
-                :cylc:conf:`suite.rc[runtime][<namespace>][remote]retrieve job logs`.
+                :cylc:conf:`suite.rc[runtime][<namespace>][remote]retrieve job
+                logs`.
```

Comment:
Damn, I remember this was something I was going to go back to later.

When you build the docs does this reference still work? The cylc domain might possibly need some tweaking to let it pass, or I might have already done this and forgotten.

Reply:
I have built it and I think it works as intended.

Reply:
```
Exception occurred:
  File "/Users/oliver/cylc-sphinx-extensions/cylc/sphinx_ext/cylc_lang/domains.py", line 118, in tokenise
    raise ValueError(f'Not a valid namespace "{namespace_string}"')
ValueError: Not a valid namespace "suite.rc[runtime][<namespace>][remote]retrieve job
logs"
```

Reply:
Simple fix: https://github.com/cylc/cylc-sphinx-extensions/pull/25

Reply:
hmmm weird I didn't get that error

Reply:
hmmm, are all three branches upto date.

Reply:
:) oopsie, hadn't updated master on cylc sphinx extensions, I'm such a lemon sometimes!

Reply:
I double checked this, managed to get that failure. Now working well with other pr checked out.